### ISSUE TITLE
SYNC field

### ISSUE THREAD

**Issue by sp5wwp** [M17-Project/M17_spec#7]
There should be two different SYNCs:
one for the initial LICH (can be first digits of e, that is 0x2B7E)
one for all other frames (we already have pi digits 0x3243 there, so OK)

**Comment by elms** [comment]
@sp5wwp Why do we want two different syncs? Was there a discussion that you can link?

**Comment by elms** [comment]
Background...This was discussed previously, but lost to the ether. Recap from today:
> <~SP5WWP> elms, its because it'd be a lot easier to compare sync fields than to convert type-4 bits to type-1 and check if the CRC is right :)
> <~SP5WWP> just to see if the rcv'd frame is LICH frame or data frame
> <%elms> Can you summarize on the github? ...I'm not sure I follow that
> <~SP5WWP> different syncs help in differentiating initial LICH and later data frames
> 

My 2 cents:
I think there is probably a better approach, but more invasive. I consider `sync` layer 1 and type of data to be layer 2. We should aim to keep them separate.

**Comment by mach327** [comment]
(more discussion is at https://forum.m17project.org/showthread.php?tid=36 )

**Comment by sp5wwp** [comment]
We could change syncwords back to 0x3243 and 0x98E9 as its complement.
